gradlew

```diff
@@ -175,14 +175,9 @@ save () {
     for i do printf %s\\n "$i" | sed "s/'/'\\\\''/g;1s/^/'/;\$s/\$/' \\\\/" ; done
     echo " "
 }
-APP_ARGS=$(save "$@")
```

Comment:
Whence this change? Are we applying some sort of shell linter? (I'm curious because I have never seen backticks recommended over `$()`)

Reply:
It's a change from Gradle itself. I simply ran `./gradlew wrapper --gradle-version 6.2`.